.github/dependabot.yml

```diff
@@ -9,21 +9,83 @@ updates:
     schedule:
       interval: "daily"
     reviewers:
-      - "hixie"
-      - "godofredoc"
       - "keyonghan"
       - "caseyhillers"
     labels:
       - "team"
       - "team: infra"
       - "waiting for tree to go green"
   - package-ecosystem: "pub"
-    directory: "/"
+    directory: "/test_utilities"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "keyonghan"
+      - "caseyhillers"
+    labels:
+      - "team"
+      - "team: infra"
+      - "waiting for tree to go green"
```

Comment:
Can we start using `autosubmit` instead?

Reply:
> Can we start using `autosubmit` instead?

Done!